ITEMS.md

```diff
@@ -0,0 +1,147 @@
+
+Let's take, for example, the best farming item in KOL history -- the Robortender. At writing, the Robortender can be had for 750 million meat. Having a Robortender will make you (approximately) 300,000 more meat per day over someone whose best meat drop familiar is a Hobo Monkey. That's quite a lot, and (again) is by a pretty considerable margin the best farming item in the game. But at a price like 750 million, the time it would take to recoup your investment from Barf farming is absolutely ridiculous. It would take you a startling **2,500** days of farming to recoup that expenditure -- seven years straight of your KOL account doing nothing but farming. _Jeez._
+
+So, to reiterate: please, _do not_ take this as a "suggested IOTMs to buy" list. This is simply a list of what Garbo supports. If you have an IOTM that is not included in this list, but you think it -should-, let us know in the #garbage-collector channel on the ASS Discord, and we can walk you through the process of submitting a pull request. Thank you for reading, and happy garbage farming!
+
+| Item                                    | Type               | Notes                                                                              |
+| --------------------------------------- | ------------------ | ---------------------------------------------------------------------------------- |
+| Beach Comb                              | Accessory          | Familiar weight beachcomb buff                                                     |
+| Guzzlr tablet                           | Accessory          | Garbo uses Guzzlr to extract a bit of value from wanderers, by gaining guzzlrbucks |
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
+| mint-in-box Powerful Glove              | Accessory          | Monster Replacer                                                                   |
+| packaged backup camera                  | Accessory          | Copier, +Meat Drops                                                                |
+| Unopened Eight Days a Week Pill Keeper  | Accessory          | Used to double the next potion.                                                    |
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                                            |
```

Comment:
guaranteed crit source for the pointer ring

Reply:
resolved in new PR